openedx/features/genplus_features/genplus/models.py

```diff
@@ -111,6 +111,10 @@ def __str__(self):
         else:
             return str(self.gen_user.pk)
 
+    @property
```

Comment:
please use related_name='classes' in manytomanyfield instead

Reply:
nevermind shavaiz added related name in ClassStudents in #73

Reply:
I will be rebasing